testing/py-atomicwrites/APKBUILD

```diff
@@ -0,0 +1,39 @@
+##
+# Copyright (c) 2018 Riverbed Technology, Inc.
```

Comment:
We usually don't do Copyright headers in APKBUILDs. Not sure how to handle this in this case…

Reply:
Thanks for the note...  Should I pull it out and re-submit?

Reply:
Idk, if this is work you've done for you employer it might be necessary to indicate that somehow? I am not a copyright lawyer though :sweat_smile:

Reply:
It is "recommended", if it will help to pytest working again, I can re-submit without the headers...

Reply:
It rases a couple of interesting questions.

What license are aports under?

Is the copyright handed over to Alpine Linux project when contributing?

I might have missed in my search but it should properly be mentioned on https://wiki.alpinelinux.org/wiki/Alpine_Linux:Contribute, here on GitHub and elsewhere.

Maybe @clandmeter @ncopa can answer this?